certbot/client.py

```diff
@@ -53,9 +54,15 @@ def determine_user_agent(config):
     """
 
     if config.user_agent is None:
-        ua = "CertbotACMEClient/{0} ({1}) Authenticator/{2} Installer/{3}"
-        ua = ua.format(certbot.__version__, util.get_os_info_ua(),
-                       config.authenticator, config.installer)
+        if cli.cli_command in ["certbot", "letsencrypt", "certbot-auto", "letsencrypt-auto"]:
+            auto = cli.cli_command
+        else:
+            auto = "other"
```

Comment:
(I think the benefits would be positive but extremely small, essentially limited to cases where a `certbot-auto` user sets `VENV_NAME` or `XDG_DATA_HOME`)